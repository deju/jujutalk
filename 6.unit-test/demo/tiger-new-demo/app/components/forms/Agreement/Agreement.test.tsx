import React from 'react';
import { render } from '@testing-library/react';
import Agreement from './';

test('CHN snapshot', () => {
    const { asFragment } = render(<Agreement country="CHN" />);

    expect(asFragment()).toMatchSnapshot();
});

test('USA snapshot', () => {
    const { asFragment } = render(<Agreement country="USA" />);

    expect(asFragment()).toMatchInlineSnapshot(`
        <DocumentFragment>
          <div>
             this is for USA 
          </div>
        </DocumentFragment>
    `);
});
