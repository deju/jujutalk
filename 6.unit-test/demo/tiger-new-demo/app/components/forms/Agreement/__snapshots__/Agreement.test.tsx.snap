// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CHN snapshot 1`] = `
<DocumentFragment>
  <div>
     this is for china 
  </div>
</DocumentFragment>
`;
